KsApi/Service.swift

```diff
@@ -549,7 +549,7 @@ public struct Service: ServiceType {
     let query = GraphAPI
       .FetchProjectRewardsByIdQuery(
         projectId: projectId,
-        includeShippingRules: false,
+        includeShippingRules: true,
```

Comment:
Tested around the app to make sure this doesn't impact anything unexpectedly. All this does is include shipping rule/location data in each reward for a project. 

In theory, this might cause a slower project load time. I haven't seen a notable difference, but I will keep an eye on it.